files/backends.py

```
@@ -43,11 +44,44 @@ def _check_returncode(self, process):
+
+        try:
+            # First try graceful termination
+            if self.process.poll() is None:  # Check if still running
+                logger.info("Terminating FFmpeg process gracefully")
+                self.process.terminate()
+                
+                # Wait up to 5 seconds for graceful shutdown
+                try:
+                    self.process.wait(timeout=5)
+                    logger.info("FFmpeg process terminated gracefully")
+                except:
```

Ruff (0.14.3)

Do not use bare `except`

(E722)